docs/getting-started/app-custom-theme.md

```diff
@@ -228,6 +228,88 @@ const myTheme = createUnifiedTheme({
 });
 ```
 
+## Custom Fonts
+To add custom fonts, you first need to declare the font style following the `@font-face` syntax from [Mui Typography](https://mui.com/material-ui/customization/typography/). After that you can then utilize the `styleOverrides` of `MuiCssBaseline` under components to set the default font. 
+```ts title="packages/app/src/theme/myTheme.ts"
+import MyCustomFont from '../assets/fonts/My-Custom-Font.woff2';
+
+const myCustomFont = {
+  fontFamily: 'My-Custom-Font',
+  fontStyle: 'normal',
+  fontDisplay: 'swap',
+  fontWeight: 300,
+  src: `
+    local('My-Custom-Font'), 
+    url(${MyCustomFont}) format('woff2'), 
+  `,
+},
+
+export const myTheme = createUnifiedTheme({
+  fontFamily: 'My-Custom-Font',
+  components: {
+    MuiCssBaseline: {
+      styleOverrides: {
+        '@font-face': [myCustomFont],
+      },
+    },
+  },
+};
+```
+If you want to utilize different fonts, then you can set the top level `fontFamily` to what you want for your body, and then override `fontFamily` in `typography` to control fonts for various headings. 
```

Comment:
We probably want a space above this line

Reply:
added space